src/test/java/org/apache/commons/compress/compressors/lz4/FramedLZ4CompressorRoundtripTest.java

```diff
@@ -104,15 +104,18 @@ private void roundTripTest(final String testFile, final FramedLZ4CompressorOutpu
 
     @Test
     public void test64KMultipleBlocks() throws IOException {
-        final ByteArrayOutputStream buffer = new ByteArrayOutputStream();
-        final byte[] expected = new byte[98304];
+        ByteArrayOutputStream buffer = new ByteArrayOutputStream();
```

Comment:
Hello @cpreisler 
Thank you for your update.
Why has this method changed? I can't spot the functional differences. Why is `count` declared but not used? Revert if this is just formatting or noise.

Reply:
You are correct. There was no functional changes. I guess I got a little confused during the initial commit of my changes today. I did remove the unused count variable. I will be more careful with my merge/commits in the future.